src/main/kotlin/org/oso/core/events/InvitationEvent.kt

```diff
@@ -0,0 +1,13 @@
+package org.oso.core.events
+
+import org.oso.core.entities.HelpProvider
+import org.oso.core.entities.HelpRequester
+import org.springframework.context.ApplicationEvent
+import java.util.*
+
+data class InvitationEvent(
+    val helpRequester: HelpRequester,
+    val helpProvider: HelpProvider,
+    val baseUrl: String,
```

Comment:
Is it possible to use the URL class directly here instead of a simple String?

Reply:
Yes, I'll change that.